source/features/highest-rated-comment.tsx

```diff
@@ -68,7 +68,7 @@ function linkBestComment(bestComment: HTMLElement): void {
 	}
 
 	const text = select('.comment-body', bestComment)!.textContent!.slice(0, 100);
-	const {hash} = select('a.js-timestamp', bestComment)!;
+	const {hash} = select<HTMLAnchorElement>('a.js-timestamp', bestComment)!;
```

Comment:
@g-plane do you know what's happening here? Suddenly `select-dom` can't parse an `'a'` selector into `HTMLAnchorElement`;  It doesn't even return `HTMLElement`, but just an `Element`. `document.querySelector('a')` works correctly.

Reply:
<img width="636" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/109454478-ef26ca80-7a19-11eb-84b8-33e7b0a6fa3b.png">

Reply:
Must be a TypeScript bug. In the same file, on a different line, it works correctly:

<img width="651" alt="Screen Shot 10" src="https://user-images.githubusercontent.com/1402241/109454550-17162e00-7a1a-11eb-876f-f6bcaabd7379.png">

Reply:
Apparently TypeScript just does not like that `const {hash}`, because if I change it, it works correctly:

<img width="615" alt="Screen Shot 11" src="https://user-images.githubusercontent.com/1402241/109454716-707e5d00-7a1a-11eb-9148-963488e1ccc7.png">

Sorry for bothering you, it's a TypeScript bug.